src/common/box_filters.c

```diff
@@ -1279,7 +1409,12 @@ static inline float window_max(const float *x, int n)
 
 // calculate the one-dimensional moving maximum over a window of size 2*w+1
 // input array x has stride 1, output array y has stride stride_y
-static inline void box_max_1d(int N, const float *const restrict x, float *const restrict y, size_t stride_y, int w)
+static inline void box_max_1d(
+    int N,
```

Comment:
N, stride_y and w could maybe be consts

Reply:
Done.  I realized after the first couple of files that I shouldn't be reformatting everything.